libgoal/participation.go

```diff
@@ -155,6 +155,12 @@ func (c *Client) GenParticipationKeysTo(address string, firstValid, lastValid, k
 	if err != nil {
 		return
 	}
+	_, err = os.Stat(partKeyPath)
+	if !os.IsNotExist(err) {
+		err = fmt.Errorf("ParticipationKeys exist for the range %d to %d", firstRound, lastRound)
```

Comment:
I suppose if `err == nil` then the file necessarily exists and we can return, and if `err!= nil` (whatever error that might be) we should just ignore it (or log? I think it'll be logged later if needed) and continue the function.
However, I need to consult @algoidan as I'm not sure if this takes into consideration the case where the file itself is corrupted (by the process of writing to it being terminated for example).

The reason for this change was mostly performance since generating the falcon keys takes around 8 minutes (if I recall correctly), but maybe the simplest solution is the best - just override whatever there is if it exists.

Reply:
so, that's not what I meant. I was thinking about this:
```golang
	_, err = os.Stat(partKeyPath)
	if !os.IsNotExist(err) {
		if err != nil {
			err = fmt.Errorf("participation key file '%s' cannot be accessed : %w", partKeyPath, err)
		} else {
			err = fmt.Errorf("ParticipationKeys exist for the range %d to %d", firstRound, lastRound)
		}
		return
	}
```